Add XMLSerializer() docs (#25619)
mdn/content: files/en-us/web/api/xmlserializer/index.md

### Serializing XML into a string

This example just serializes an entire document into a string containing XML.

```js
 const s = new XMLSerializer();
 const str = s.serializeToString(document);
 saveXML(str);
```

> This involves creating a new `XMLSerializer` object, then passing the {{domxref("Document")}} to be serialized into {{domxref("XMLSerializer.serializeToString", "serializeToString()")}}, which returns the XML equivalent of the document. The function `saveXML()` stands for the subsequent action on the serialized string.

Reviewer: What do you mean by "stands for the subsequent ...". Something like:
```suggestion
This involves creating a new `XMLSerializer` object, then passing the {{domxref("Document")}} to be serialized into {{domxref("XMLSerializer.serializeToString", "serializeToString()")}}, which returns the XML equivalent of the document. The function `saveXML()` represents a function that would then save the serialized string.
```
